src/API/Google/Settings.php

```diff
@@ -387,6 +387,58 @@ public function get_store_address(): AccountAddress {
 		return $mc_address;
 	}
 
+	/**
+	 * Check whether the address has errors
+	 *
+	 * @param AccountAddress $address to be validated.
+	 *
+	 * @return array
+	 */
+	public function has_address_errors( AccountAddress $address ): array {
+		if ( ! $address instanceof AccountAddress ) {
+			return [];
```

Comment:
❓  So if we send something wrong to `has_address_errors` is returns [], hence no error?

Reply:
Also this condition is always 'false' because 'AccountAddress $address' is evaluated at this point because of the types

Reply:
Yes, you are right it is unnecessary as PHP will throw an error before evaluating that condition. 

Has been removed in [7b1c89b](https://github.com/woocommerce/google-listings-and-ads/pull/1169/commits/7b1c89bf566dfd9c097f84ac6e9d256a54f7b584)